removal_from_ansible.rst

```diff
@@ -119,8 +119,19 @@ Process
 Unmaintained collections
 ========================
 
-Identifying and removing an unmaintained collection
----------------------------------------------------
+Collections that have been announced deprecated / unmaintained by their maintainers
+-----------------------------------------------------------------------------------
+
+Process
+~~~~~~~
+
+#. `Announce upcoming removal from the next major Ansible release <announce_removal_>`_.
+#. `Remove collection from the next major Ansible release <remove_collection_>`_.
```

Comment:
Picking a corner case here - What if a collection owner decides it's deprecated/unmaintained late in an Ansible development cycle?  Say foo.bar collection is deprecated a week before the Ansible 10 alpha - do we still remove it in Ansible 10?

Reply:
@samccann Good point, we should clarify this.

In this case, I think we should remove the collection since it's only an alpha release. That's not late, it's early in the development cycle. But probably not once there's a pre-release that comes with a feature freeze (Ansible-10.0.0 beta1).

What do you think about:

```
If the current major release is X and there hasn't been a feature freeze release of the next major version X+1:

#. `Announce upcoming removal from the (X+1).0.0 Ansible release <announce_removal_>`_.
#. `Remove collection from the (X+1).0.0 Ansible release <remove_collection_>`_.

If there already has been a feature freeze release of the next major version X+1:

#. `Announce upcoming removal from the (X+2).0.0 Ansible release <announce_removal_>`_.
#. `Remove collection from the (X+2).0.0 Ansible release <remove_collection_>`_.
```

Reply:
Or simply define a new variable `Y` which is `X+1` or `X+2`, and then use `Y`, to avoid having the list of instructions twice :)

Reply:
@samccann Or, as @felixfontein suggested, simplify this using a variable `Y`:

```
If the current major release is X and there hasn't been a feature freeze release of the next major version X+1, remove the collection from Y=(X+1).0.0.
If there already has been a feature freeze release of the next major version X+1, remove the collection from Y=(X+2).0.0.

#. `Announce upcoming removal from the (Y Ansible release <announce_removal_>`_.
#. `Remove collection from the Y Ansible release <remove_collection_>`_.
```

Reply:
There's now an opening bracket in the first `#.` line that shouldn't be there, but besides from that, LGTM.